src/background.js

```diff
@@ -58,7 +68,12 @@ chrome.runtime.onMessageExternal.addListener(function(request, sender) {
     }
 
     // generate code
-    copyToClipboard(otp.generate());
+    var code = otp.generate();
+
+    // copy to clipboard
+    if (!request.action.match(/^copy[A-Z]*/)) {
```

Comment:
maybe just use "startsWith" ?

Reply:
`startsWith()` doesn't take a regex though.

Reply:
Ah, I see, you wanted to highlight the need for a capital letter

Reply:
Yup.